### DISCUSSION TITLE
does nuxt support grpc or not ?

### DISCUSSION THREAD

**Discussion by xdewx** [nuxt/nuxt#16883] [Questions]
### Describe the feature

I want to use grpc in node, there is any way to achieve it?

### Additional information

- [x] Would you be willing to help implement this feature?
- [x] Could this feature be implemented as a module?

### Final checks

- [X] Read the [contribution guide](https://v3.nuxtjs.org/community/contribution).
- [X] Check existing [discussions](https://github.com/nuxt/framework/discussions) and [issues](https://github.com/nuxt/framework/issues).

**Comment by sianami123** [comment]
i want use grpc-web via envoy proxy in nuxt3
any update?
